### PR TITLE
replace homepage

### PR DESCRIPTION
**Pull request by manfordbenjamin**
The homepage looked boring so made it look a little attractive

### CODE REVIEW

**Comment by alkchr** [comment]
:+1: With this change the home page looks better indeed.

## Header
### Before
header_before
### After
header_after

## Body
### Before
body_before
### After
body_after

P.S.: In subsequent iterations it would be a good idea to move `styles="..."` parameters to a CSS file.

**Comment by manfordbenjamin** [comment]
Sure. Will do

**Comment by alkchr** [comment]
ping @pedromorgan @brendensoares

File: index.html [hunk 1]
```diff
@@ -26,19 +26,26 @@
...
+                <div class="col-md-8">
+                    <h1>Revel</h1>
+                    <small style="font-size: 14pt;">A high-productivity web framework for the Go language.</small>
+                    
+                    <br /><br /><br />
+
+                    <div style="font-size: 14pt;">
+                        <small>Latest Release:</small><br><br>
+                        <b><a href="https://github.com/revel/revel/releases">v{{site.data.version.revel}}</a></b> <br>
+                        <small>{{site.data.version.date}} - Daffodil release</small><br>
```
Comment (brendensoares) [inline index.html]: Please remove "Daffodil release" and we'll merge this ASAP. Nice enhancement, btw 💃

**Comment by jeevatkm** [comment]
Manually targeted and merged to `develop` branch.

**Pull request closed without merging**